src/test/clojure/clara/test_accumulators.clj

```diff
@@ -284,3 +284,48 @@
     (is (= 1 (count temp-history)))
     (is (= [{:?his (->TemperatureHistory [temp-10-mci])}]
             temp-history))))
+
+(deftest test-grouping-accum
+  (let [grouping-accum (acc/grouping-by :temperature)
+        grouping-query (dsl/parse-query [] [[?t <- grouping-accum
+                                             :from [Temperature]]])
+
+        convert-return-fn (fn [m]
+                            (if (seq m)
```

Comment:
I guess I was more curious about whether or not `convert-return-fn` could be a bit smarter and do this by default. Too much?